### DISCUSSION TITLE
Self hosting on Railway

### DISCUSSION THREAD

**Discussion by cmaclennan** [Budibase/budibase#9274] [Help]
Just curious if anyone has tried self hosting on Railway-App? Or might know how to get it to deploy, similar to Digital Ocean.

TIA

**Comment by melohagan** [comment]
Hey @cmaclennan 

I haven't deployed Budibase on Railway myself, however at a quick glance of the docs I see that it supports [Dockerfiles](https://docs.railway.app/deploy/dockerfiles)

Docker-compose would have been easier, but you may still be able to make it work by having a look at the implementation of our single image install: https://github.com/Budibase/budibase/tree/develop/hosting/single

Maybe you can get the Dockerfile there to work.

**Comment by WayneSimpson** [comment]
> Just curious if anyone has tried self hosting on Railway-App? Or might know how to get it to deploy, similar to Digital Ocean.
> 
> TIA

Did you manage to get it up on Railway?
